doc/dynamo0p3.rst

```diff
@@ -429,20 +429,26 @@ operation. In this case:
 Rules for Inter-Grid Kernels
 ++++++++++++++++++++++++++++
 
-1) An inter-grid kernel is identified by the presence of a field argument with
-   the optional `mesh_arg` meta-data element (see
+1) An inter-grid kernel is identified by the presence of a field or
+   field-vector argument with the optional `mesh_arg` meta-data element (see
    :ref:`dynamo0.3-intergrid-mdata`).
 
-2) An inter-grid kernel is only permitted to have field or field-vector
+2) An invoke that contains one or more inter-grid kernels must not contain
```

Comment:
Yes, it is a restriction made to keep the implementation simple. I checked this with @christophermaynard and he said:

"Mixing them probably has no benefit as one cannot fuse because loop orders are going to be different. Fine to Ban."

I've updated the documentation to make it clear that this is an implementation issue and that the restriction could be lifted in future work if it becomes desirable.